common/src/types/tasks/descriptors/mod.rs

```diff
@@ -229,7 +228,7 @@ pub mod mocks {
 
         // Serialize the commitment, uses the contract's serialization here:
         //  https://github.com/renegade-fi/renegade-contracts/blob/main/contracts-common/src/custom_serde.rs#L82-L87
-        let comm_bytes = new_wallet_comm.inner().serialize_to_bytes();
+        let comm_bytes = new_wallet_comm.to_bytes_be();
```

Comment:
as in `update_wallet.rs`, can you update the comment here to specify that we are matching, not using, the serialization used in the contracts